src/setup/index.ts

```diff
@@ -30,18 +30,18 @@ const cwd = process.cwd();
       validate: (val: string) => val.trim().split(' ').shift()!.length > 0,
     }]);
 
-  let accountInfo = null;
-  if (serverUrl === 'wss://s.altnet.rippletest.net:51233') {
-    console.log('▲ Generating Testnet account credentials...');
-    accountInfo = await got.post('https://faucet.altnet.rippletest.net/accounts', {json: true})
-      .then(({body}) => body.account)
-      .catch((err) => console.log('  ' + err.message));
-  } else if (serverUrl === 'wss://s.devnet.rippletest.net:51233') {
-    console.log('▲ Generating Devnet account credentials...');
-    accountInfo = await got.post('https://faucet.devnet.rippletest.net/accounts', {json: true})
+  const network = serverUrl === 'wss://s.altnet.rippletest.net:51233' ? 'altnet' : serverUrl === 'wss://s.devnet.rippletest.net:51233' ? 'devnet' : 'other';
```

Comment:
This is a bit confusing to me. I get what's going on, but it's a bit hard to parse with two different ternarys happening in the same line.

Reply:
So would _if - else if - else_ be better?